library/jsonviewer/src/main/java/org/billcarsonfr/jsonviewer/JSonViewerDialog.kt

```diff
@@ -27,9 +27,9 @@ import com.airbnb.mvrx.Mavericks
 class JSonViewerDialog : DialogFragment() {
 
     override fun onCreateView(
-        inflater: LayoutInflater,
-        container: ViewGroup?,
-        savedInstanceState: Bundle?
+            inflater: LayoutInflater,
```

Comment:
it looks like these formatting changes could be done separately from the parameter changes (would also help reduce the noise is we're aiming to incrementally change the formatting rules)

Reply:
I tried isolating this change as much as possible, but I found that we can't do "I only want to reformat this change". Even if you removed all other lines from editorconfig, the IDE wants to follow each rule as either true or false.

In this case, this is a case of us having continuation indents enabled. If we disabled it, the rest of the project that is using continuation indents will change.

This is a consequence of us having inconsistent formatting throughout the project

Reply:
my bad I meant that we could format the codebase without the editconfig changes in this file to get a baseline, then all the PRs afterwards will only contain the new changes

Reply:
That approach could work too, though I guess the PR for that wouldn't be this one. I'll go ahead and make a new one

Reply:
Closing this for now. Will reapproach after merging https://github.com/vector-im/element-android/pull/5805